platform/windows/export/export.cpp

```diff
@@ -69,17 +69,17 @@ Error EditorExportPlatformWindows::export_project(const Ref<EditorExportPreset>
 	}
 
 	if (!FileAccess::exists(rcedit_path)) {
-		ERR_PRINTS("Could not find rcedit executable at " + rcedit_path + ", aborting.");
-		return ERR_FILE_NOT_FOUND;
+		ERR_PRINTS("Could not find rcedit executable at " + rcedit_path + ", no icon will be included.");
+		return OK;
```

Comment:
This will make it skip the `codesign` part at the end which is independent from the use of rcedit.

Reply:
Though that's already the case currently due to the `rcedit_path == String()` early return, so that's not adding a bug.

Maybe move the whole rcedit stuff to a dedicated private function? This way you can still make it return `OK` while continuing the `export_project` logic to do code signing.

Reply:
Done.